docs/snippets/ov_multi.py

```diff
@@ -81,18 +83,18 @@ def available_devices_2():
 def MULTI_4():
 #! [MULTI_4]
     core = Core()
-    hddl_config = {}
+    cpu_config = {}
     gpu_config = {}
 
     # Read a network in IR, PaddlePaddle, or ONNX format:
     model = core.read_model(model_path)
 
-    # When compiling the model on MULTI, configure CPU and MYRIAD 
+    # When compiling the model on MULTI, configure CPU and GPU 
     # (devices, priorities, and device configurations):
     core.set_property(device_name="GPU", properties=gpu_config)
-    core.set_property(device_name="HDDL", properties=hddl_config)
-    compiled_model = core.compile_model(model=model, device_name="MULTI:HDDL,GPU")
-    
+    core.set_property(device_name="CPU", properties=cpu_config)    
```

Comment:
Remove `core.set_property(device_name="GPU", properties=gpu_config)` and `core.set_property(device_name="CPU", properties=cpu_config)`. `gpu_config` and `cpu_config` will load during `compile_model()`